pyqgisserver/handlers/statushandler.py

```diff
@@ -34,9 +35,15 @@ def head(self):
 
 class StatusHandler(BaseHandler):
 
+    def initialize(self) -> None:
+        super().initialize()
+        config = confservice['management']
+        self.management_proxy_url = config['proxy_url'].strip('/')
+
     def get(self):
 
         path = self.request.path.rstrip('/')
+        management_proxy_url = self.management_proxy_url
```

Comment:
This handler may be used also in the exposed api so you cannot handle  the managment url here. 
The managment server support xheaders and `forwarded-for` headers, use this in your reverse-proxy in order to get the correct host url (don't to change the path)